source/Contribution-Guide/guide-examples/maintenance-guide.html.md.erb

```diff
@@ -0,0 +1,41 @@
+---
+title: Maintenance Guide Template
+last_reviewed_on: 2025-02-25
+review_in: 6 months
+weight: 10
+---
+
+# <%= current_page.data.title %>
+
+## [System/Application Name] Maintenance Guide
+This guide outlines routine maintenance tasks for [System/Application Name] to ensure optimal performance and reliability.
+
+## Prerequisites
+- Administrative access to [System/Application Name]
+- Scheduled maintenance window
+- Backup of current system state
+
+## Maintenance Tasks
+
+### Task 1: [Brief description of maintenance task]
+Frequency: [How often this task should be performed]
+1. Step one
+2. Step two
+3. Step three
```

Comment:
These steps aren't being rendered correctly for some reason. The other templates are ok though

Reply:
cheers - sorted, damn spacing